### DISCUSSION TITLE
More icons

### DISCUSSION THREAD

**Discussion by shinebrillant** [enrique-lozano/Monekin#252] [Ideas / New features]
I hope there will be more icons and a search function.
For example, Google Material Symbols and Icons.
https://fonts.google.com/icons?icon.style=Rounded

**Comment by enrique-lozano** [comment]
HI, thanks for the suggestion!

The symbols are already obtained from the Google Fonts. If you want more you just have to add the ID of the icon to our json file, alongside with its category. Please check [this folder](https://github.com/enrique-lozano/Monekin/tree/main/lib/core/services/supported_icon/getter) for more info. After put the new icons in the json file, they should be re-generated via the python script that it's in this folder.

The ID of the icon can be obtained from the Google fonts page:

![image](https://github.com/user-attachments/assets/76264c2c-ae0f-4736-b92b-d29914deae22)


I will consider any new icon suggestion that you put me inside any discussion, and of course, any PR with the new icons in the json file

The search icon would be a great feature I think, but not my priority at this moment

**Comment by enrique-lozano** [comment]
+25 more new icons added in #289
